org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/org.OpenT2T.Sample.SuperPopular.Thermostat-2.0.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.OpenT2T.Sample.SuperPopular.Thermostat">
+    <description>OpenT2T SuperPopular Thermostat interface</description>
+
+    <property name="PowerStatus" type="s" access="readwrite">
```

Comment:
I guess we should be following AllJoyn notation in the XML. corrrect ?

[https://wiki.allseenalliance.org/irb/interface_design_guidelines_1.0](https://wiki.allseenalliance.org/irb/interface_design_guidelines_1.0)

Member Names
Member (property, signal and method) names must use UpperCamelCase notation without punctuation

---
In reply to: [73768119](https://github.com/openT2T/translators/pull/37#discussion_r73768119) [](ancestors = 73768119)